packages/falso/src/lib/phone-number.ts

```diff
@@ -12,15 +246,36 @@ import { randNumber } from './number';
  *
  * @example
  *
+ * randPhoneNumber({ countryCode: 'US' })
+ *
+ * @example
+ *
  * randPhoneNumber({ length: 10 })
  *
  */
-export function randPhoneNumber<Options extends FakeOptions>(
-  options?: Options
-) {
-  return fake(() => {
-    const number = '1' + randNumber({ min: 1_000_000_000, max: 9_999_999_999 });
+export function randPhoneNumber<
+  Options extends FakeOptions & {
+    countryCode?: CountryCode;
+  }
+>(options?: Options) {
+  let formats = data.map(({ formats }) => formats).flat();
+
+  if (options?.countryCode) {
+    formats =
+      data.find((country) => {
+        return country.countryCode.includes(options.countryCode!);
+      })?.formats || [];
+  }
+
+  const minNumb = 0;
+  const maxNumb = 9;
 
-    return number.replace(/1(.{3})(.{3})(.{4})/, '1-$1-$2-$3');
-  }, options);
+  let phoneNumber = Array.from({ length: options?.length || 1 }, (_, index) => {
+    return rand(formats).replace(/#/g, () => {
+      return (
+        '' + (Math.floor(Math.random() * (maxNumb - minNumb + 1)) + minNumb)
+      );
+    });
+  }) as any;
```

Comment:
why as any?

Reply:
My mistake, I must have sensed it when using it in randUser 😁